### ISSUE TITLE
Spacing and bullets

### ISSUE THREAD

**Issue by jean-gui** [w3c/w3c-website#680]
Recent CSS changes have had some unexpected side-effects related to bullets in lists and spacing:

- Bullets are now displayed in front of some lists:
  - "Chairs" and "Staff Contacts" on group pages. Those lists are also misaligned with their "Chairs" and "Staff Contacts" titles
    - See: https://www.w3.org/groups/wg/apa/
    - Before: https://web.archive.org/web/20240917131212/https://www.w3.org/groups/wg/apa/
    - Related: https://github.com/w3c/w3c-website/issues/679
  - Ecosystem testimonial slider
    - See: https://www.w3.org/ecosystems/publishing/#slider-title
    - Before: https://web.archive.org/web/20241008054454/https://www.w3.org/ecosystems/publishing/
  - Autocomplete fields
    - See the "mailing list" field on https://www.w3.org/Search/Mail/Public/search or "Countries" on https://www.w3.org/Member/ACList
    - There is also some space between the field and the list of results that appears below, it didn't use to
  - Webhooks list
    - See: https://www.w3.org/users/myprofile/webhooks/ (@NicolaSaunders I've created a webhook for you)
  - Webhooks view
    - See: from https://www.w3.org/users/myprofile/webhooks/ edit a webhook
- No spacing anymore inside the Ecosystem evangelist card
  - See: https://www.w3.org/ecosystems/media/
  - Before: https://web.archive.org/web/20241008054438/https://www.w3.org/ecosystems/media/
- Spacing issue between mailing lists entries
  - See: https://lists.w3.org/Archives/Public/
  - Before: https://web.archive.org/web/20241007110923/https://lists.w3.org/Archives/Public/
- In the "Related RSS feed" and "API" blocks both the anchor and icons are displayed on a new line
  - See: https://www.w3.org/groups/wg/apa/
  - Before: https://web.archive.org/web/20240917131212/https://www.w3.org/groups/wg/apa/

**Comment by NicolaSaunders** [comment]
I have raised [Pull Request 162](https://github.com/w3c/w3c-website-templates-bundle/pull/162) which tackles the following:

- Unwanted bullets on the content slider controls
- Unwanted bullets on the autocomplete field
- Lack of spacing on in Enavgelist cards

> Bullets are now displayed in front of some lists

The `.list-no-bullets` only worked because of a more general rule which removed the default bullets and padding from a list with any class. That class is misleading really - people might expect the `.list-no-bullets` class to be doing something directly, but it doesn't exist in the design system. To restore the old visual appearance it should be replaced with the `.clean-list` class, which has explicit styles assigned to it in the design system. That will need addressing in the markup where appropriate.

> Spacing issue between mailing lists entries

There is some CSS that has been added directly to this page using `<style>`. One of the declarations there is:

```
.clean-list.lol li {
  padding-top: 1em;
  padding-bottom: 1em;
}
```

I think this needs to be changed to the following:

```
.clean-list.lol > li {
  padding-top: 1em;
  padding-bottom: 1em;
}
```

That will prevent the additional padding from being added to items in the nested lists
